src/types/index.ts

```diff
@@ -242,6 +246,15 @@ export interface EnableGeneralPermissionManagerProcedureArgs {
   symbol: string;
 }
 
+export interface EnableGeneralTransferManagerProcedureArgs {
+  symbol: string;
+}
+
+export interface DisableFeatureArgs {
```

Comment:
Should be called `DisableFeatureProcedureArgs`

Reply:
Fixed!